generators/chipyard/src/main/scala/TileOnlyConfig.scala

```diff
@@ -0,0 +1,210 @@
+        executable = true,
+        supportsArithmetic = TransferSizes(1, beatBytes),
+        supportsLogical = TransferSizes(1, beatBytes),
+        supportsGet = TransferSizes(1, 64),
+        supportsPutFull = TransferSizes(1, 64),
+        supportsPutPartial = TransferSizes(1, 64),
+        supportsHint = TransferSizes(1, 64),
+        supportsAcquireB = TransferSizes(64, 64),
+        supportsAcquireT = TransferSizes(64, 64),
+        fifoId = Some(0))),
+    beatBytes = beatBytes,
+    endSinkId = 1,
+    minLatency = 1
+  )))
+
+  masterNode :=* tile.masterNode
+
+  println(tile.masterNode.edges)
+
+  val slaveNode = TLClientNode(Seq(TLMasterPortParameters.v1(Seq(TLClientParameters(
+     name = "my-client",
+     sourceId = IdRange(0, 4),
+     requestFifo = true,
+     visibility = Seq(AddressSet(0x10000, 0xffff)))))))
+
+  tile.slaveNode := slaveNode
+
+  val intSinkNode = IntSinkNode(IntSinkPortSimple())
+  intSinkNode := tile.intOutwardNode
+
+  val intSourceNode = IntSourceNode(IntSourcePortSimple())
+  tile.intInwardNode := intSourceNode
+
+  val haltSinkNode = IntSinkNode(IntSinkPortSimple())
+  haltSinkNode := tile.haltNode
+
+  val ceaseSinkNode = IntSinkNode(IntSinkPortSimple())
+  ceaseSinkNode := tile.ceaseNode
+
+  val wfiSinkNode = IntSinkNode(IntSinkPortSimple())
+  wfiSinkNode := tile.wfiNode
+
+  val hartIdSource = BundleBridgeSource(() => UInt(2.W))
+  tile.hartIdNode := hartIdSource
+
+  val resetSource = BundleBridgeSource(() => UInt(18.W))
+  tile.resetVectorNode := resetSource
+
+  val nmiSource = BundleBridgeSource(() => new NMI(18))
+  tile.nmiNode := nmiSource
+
+
+  val tileMasterIO     = InModuleBody { println("inmodulebody"); masterNode.makeIOs() }
+  val tileSlaveIO      = InModuleBody { slaveNode.makeIOs() }
+  val intSinkIO        = InModuleBody { intSinkNode.makeIOs() }
+  val intSourceIO      = InModuleBody { intSourceNode.makeIOs() }
+  val haltSinkIO       = InModuleBody { haltSinkNode.makeIOs() }
+  val ceasesSinkIO     = InModuleBody { ceaseSinkNode.makeIOs() }
+  val wfiSinkIO        = InModuleBody { wfiSinkNode.makeIOs() }
+  val hartIdSourceIO   = InModuleBody { hartIdSource.makeIOs() }
+  val resetVecSourceIO = InModuleBody { resetSource.makeIOs() }
+  val nmiSourceIO      = InModuleBody { nmiSource.makeIOs() }
+
+  override lazy val module = new TileOnlyDigitalTopImp(this)
+}
+
+class TileOnlyDigitalTopImp(val outer: TileOnlyDigitalTop)(implicit p: Parameters) extends LazyModuleImp(outer) {
+  val rocket_tile = outer.tile.module
+
+  val (master, edge) = outer.tile.masterNode.out.head
+  println(outer.tile.masterNode.outward)
+  println(master)
+  println(edge)
+}
+
+class WithTileOnlyConfig extends Config((site, here, up) => {
+  case BuildSystem => (p: Parameters) => new TileOnlyDigitalTop()(p)
+})
+
+class WithRawRocketTileConfig extends Config((site, here, up) => {
+  case RocketTileOnly => RocketTileParams(
+    beuAddr = Some(BigInt("4000", 16))
+  )
+  case XLen => 32
```

Comment:
Pls no 32b cores....

You can just disable the beu unit by setting `beuAddr = None`